gologger.go

```diff
@@ -1,6 +1,4 @@
-package gologger
-
-import "github.com/sadlil/gologger/logger"
+package main
```

Comment:
Can't use package main. By that we can't import it from other package as lib.

Reply:
No, It was fixed. It was some wrong commit with it. Look at whole changes in PR. But, I'll do it by one commit.